### ISSUE TITLE
[v1.1][New Feature] Support Ansible core 2.11

### ISSUE THREAD

**Issue by Tomorrow9** [vmware/ansible-vsphere-gos-validation#29]
##### SUMMARY

ansible-core 2.11 released on 2021-04-26,
https://docs.ansible.com/ansible-core/devel/roadmap/ROADMAP_2_11.html#

We need to test with it.

##### ISSUE TYPE
- Feature Idea

##### ENVIRONMENT / OS

All

**Comment by Tomorrow9** [comment]
```
[DEPRECATION WARNING]: [defaults]callback_whitelist option, normalizing names to new standard, use callback_enabled instead. This feature will be removed from ansible-core in version 2.15. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
```

**Comment by keirazhang** [comment]
Hit below issue with ansible 2.11:
```
redirecting filter ansible.builtin.ipv4 to ansible.netcommon.ipv4
redirecting filter ansible.builtin.ipv4 to ansible.netcommon.ipv4

TASK [Wait for VMware Tools collecting guest info] ******************************************************************
task path: /home/qiz/workspace/github/ansible-vsphere-gos-validation/common/vm_wait_guest_ip.yml:22
fatal: [localhost]: FAILED! => {
    "msg": "The conditional check 'vm_guest_facts.instance.hw_eth0.ipaddresses | ipv4' failed. The error was: template error while templating string: no filter named 'ipv4'. String: {% if vm_guest_facts.instance.hw_eth0.ipaddresses | ipv4 %} True {% else %} False {% endif %}"
}
```
The collection ansible.netcommon needs to be added into requirements.yml.

**Comment by Tomorrow9** [comment]
> ```
> [DEPRECATION WARNING]: [defaults]callback_whitelist option, normalizing names to new standard, use callback_enabled instead. This feature will be removed from ansible-core in version 2.15. Deprecation warnings can be disabled by setting deprecation_warnings=False in ansible.cfg.
> ```

This issue fixed in #80

**Comment by keirazhang** [comment]
ansible-core 2.11 is supported in https://github.com/vmware/ansible-vsphere-gos-validation/releases/tag/v1.1.
